exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -369,6 +378,10 @@ public void queryCompleted(QueryState state) {
 
     private void fail(Exception ex) {
       logger.debug("Submission failed.", ex);
+      final Throwable cause = ex.getCause();
+      if (cause != null) {
+        logger.debug("Submission failure cause.", ex.getCause());
```

Comment:
same comment related to [DRILL-2692](https://issues.apache.org/jira/browse/DRILL-2692)